basic_samples/IngressClientLibraries/DotNet/Program.cs

```diff
@@ -50,16 +53,24 @@ private static async Task MainAsync()
             string clientId = configuration["ClientId"];
             string clientSecret = configuration["ClientSecret"];
             string topicName = configuration["TopicName"];
-            string mappedClientId = configuration["MappedClientId"];
             string subscriptionName = configuration["SubscriptionName"];
+            string streamId = configuration["StreamId"];
+            string deviceClientId = configuration["DeviceClientId"];
+            string deviceClientSecret = configuration["DeviceClientSecret"];        
 
-            //Get Ingress Services to communicate with server
+            //Get Ingress Services to communicate with server and handle ingress management
```

Comment:
I would suggest "...and manage ingress" rather than "...handle ingress management"